Documentation update

kgshv opened the pull request:
Since Quix is undergoing active development, I decided to stop working on the docs passively for now, organize and submit a pull request for what I have already, get back to it once the app is more stable / is out of beta. 

This request deals with the documentation only.

frolovv commented on README.md:
```diff
@@ -1,64 +1,102 @@
-It's your single point of truth and a shared space for your company's BI insights, with quick turnaround, interactive visual perspectives, and mixed data sources.<br />
+Quix is a robust easy-to-use business intelligence & data analytics tool built to handle both quick insights and massive resource-demanding queries. It features data visualization and export, user management, and organizes queries in notebooks for easy access and sharing across teams.
+
+A web-based IDE for [Presto](https://github.com/prestodb/presto), Quix is your single point of truth and a shared space for your company's BI insights, with quick turnaround, interactive visual perspectives, and mixed data sources.
```
the correct link to presto repo is https://github.com/prestosql/presto

frolovv commented on README.md:
```diff
@@ -1,64 +1,102 @@
 
- 
-## Installation
+**N.B.** Quix relies on [Presto](http://prestodb.github.io/), which is included with the Docker container in this repository, but for the **demo purposes** only. To work with real data, it must be accessible via a Presto API URL (more on this in [Configuring](#Configuring)).
```
correct link to presto site is https://prestosql.io/

kgshv commented:
Not sure why the build would fail on this. But then again, this is outside of my area of knowledge. Just leaving this note here fyi.

kgshv commented on README.md:
```diff
@@ -1,64 +1,102 @@
+- Shared space for your company's BI insights
+- Single point of truth for all team members on public or proprietary data
+- Report generator using built-in charting tools
+- A handy tool to store, share, download, and embed live data from reports into any website (internal or public)
```
The live version of Quix (the non-OSS one) I was able to use and play around with has a feature that allows to embed a note query result.

![image](https://user-images.githubusercontent.com/4682930/57194087-2b3a9280-6f4b-11e9-9034-65df1c476e95.png)

The Quix OSS [Google Doc](https://docs.google.com/document/d/1X14-c9mLNJyySKaZbFKZmbidwL0gDsChvL5fh3SU0Mw/edit) also mentioned sharing as a feature.

Thus I expected this would be built in to the public version of Quix OSS.

@antonpod should I remove any mentions of sharing capabilities for now? Are there any plans to add the sharing feature in the future at all?

antonpod replied:
This feature is missing at the moment in OSS version, but we will add it sooner or later.

antonpod replied:
In any case, I wouldn't market it as one of the main features, as it has some gotchas.
For example, to see the embedded note, you need to be logged into Quix.

kgshv replied:
Oh, interesting. Okay, understood, thank you!

kgshv commented:
@antonpod just pushed a commit, addressing your comments and also containing some copy-editing for recently added text. Let me know if you have any questions and thank you!

antonpod commented:
@kgshv looks good, thanks.
